app/models/clinic.rb

```diff
@@ -17,13 +17,33 @@ class Clinic
   field :active, type: Boolean, default: true
   field :accepts_naf, type: Boolean, default: false
   field :gestational_limit, type: Integer
-  field :costs_9wks, type: Integer
+  field :costs_05wks, type: Integer
```

Comment:
The only reason was making it look nice and lined up. I have no preference, @Kevin-Wei liked it better with the leading zero.